easybuild/easyconfigs/p/PyTorch-extensions/PyTorch-extensions-2019.01-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-extensions'
```

Comment:
I would rename this to `PyTorch-Geometric`, since that seems to be the "flagship" project in https://github.com/rusty1s, which depends on all the other `torch-*` extension libraries being listed (see https://github.com/rusty1s/pytorch_geometric#installation)

Reply:
Agreed. Maybe if more PyTorch extensions appear we can reconsider an "extensions" module.